changelogs/unreleased/config-stuck-privilege-granting-alert.md

```diff
@@ -0,0 +1,4 @@
+## bugfix/config
+
+* An alert regarding delayed privilege granting is now cleared when the
```

Comment:
Nit: 
> privilege will be granted, when the object will be created.

privilege will be granted, when the object *is* created [1]

[1]: https://jakubmarian.com/will-after-when-in-english/

Reply:
Nit:

> dropping alerts *at* this point has no effect.

Reply:
Nit:

> The third one is implemented here with idea of code simplicity.

Reply:
All done.